src/js/views/patients/worklist/action_views.js

```diff
@@ -74,18 +74,24 @@ const ActionItemView = View.extend({
     Radio.trigger('event-router', 'patient:dashboard', this.model.get('_patient'));
   },
   onRender() {
-    this.showCheck();
-    this.showState();
-    this.showOwner();
-    this.showDueDate();
-    this.showDueTime();
     this.showForm();
     this.showDetailsTooltip();
+
+    this.canEdit = this.model.canEdit();
+
+    if (this.canEdit) {
```

Comment:
So currently the idea would be if you can't edit it then we'll show the value of these components in the template..  I think otherwise we show a read-only version of the value which might be cheaper vs having to always render the values to be replaced..  _but_ I don't think we should make the components themselves handle this state.